src/platforms/dart/configuration/integrations/logging.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Logging Integration
+caseStyle: canonical
+supportLevel: production
+sidebar_order: 3
+---
+
+The `sentry_logging` library provides [Logging](https://pub.dev/packages/logging) support for Sentry via [onRecord property](https://pub.dev/documentation/logging/latest/logging/Logger/onRecord.html) that sends events and breadcrumbs to Sentry. Once this integration is configured you can use Logging’s public API.
+
+The source can be found [on GitHub](https://github.com/getsentry/sentry-dart/tree/main/logging/lib/src).
```

Comment:
```suggestion
You can find the source on [GitHub](https://github.com/getsentry/sentry-dart/tree/main/logging/lib/src).
```

Reply:
Similar to https://github.com/getsentry/sentry-docs/pull/4799#discussion_r820745510
@imatwawana do you have a preference?

Reply:
I don't have a strong preference. Generally, it's good to use active instead of passive voice, but there's an argument for using passive voice here, so you can leave it.